mibidata/panels.py

```diff
@@ -61,11 +62,16 @@ def read_csv(path):
 
 def _merge_masses(df):
     conjugates = {}
+    targ_list = []
```

Comment:
I prefer to avoid using shortened variable names for readability, so could you rename this to `target_list`? Ask @murataksoy what happens when he tries to submit a PR for C++ code with variable names `int aaa = 0`.